src/Components/Translations.js

```diff
@@ -94,10 +94,7 @@ const translations = {
     },
   },
   fa: {
-    footer: {
-      firstLine: "محل قرار گیری فوتر",
-      secondLine: "TODO",
-    },
+    footer: "مجموعه ای از منابع مفید برای دانشجویان، ساخته شده توسط ", // TODO: translate
```

Comment:
We need one translation for the first line of the footer and one for the second line.

Reply:
please check if they are correct now